x/crosschain/keeper/msg_server_update_tss_test.go

```diff
@@ -329,3 +330,26 @@ func TestMsgServer_UpdateTssAddress(t *testing.T) {
 		require.Equal(t, len(k.GetObserverKeeper().GetSupportedChains(ctx)), len(migrators))
 	})
 }
+
+func TestKeeper_GetChainsSupportingTSSMigration(t *testing.T) {
+	t.Run("should return only ethereum and bitcoin chains", func(t *testing.T) {
```

Comment:
We could maybe use TDT for this specific one?

Reply:
I think this test covers all the scenarios already , and we can add more composite filters to that as and when needed
https://github.com/zeta-chain/zeta-node/blob/54d32d9cb2f4cf9184950b01bc1fb38bc421fde8/pkg/chains/chain_filters_test.go#L145-L150